packages/flutter_tools/lib/src/build_runner/build_script.dart

```diff
@@ -71,7 +71,8 @@ final List<core.BuilderApplication> builders = <core.BuilderApplication>[
     'flutter_tools:test_bootstrap',
     <BuilderFactory>[
       (BuilderOptions options) => const DebugTestBuilder(),
-      (BuilderOptions options) => const FlutterWebTestBootstrapBuilder(),
+      (BuilderOptions options) => FlutterWebTestBootstrapBuilder(
```

Comment:
I would avoid doing this for the test bootstrap builder if you don't have to. This creates a library which has to be compiled with DDC so it will cause a lot more duplicate work whenever the option changes.

I believe you only need to redo the entrypoint builder which for DDC is very cheap.